### PR TITLE
Fix DraggableScrollableSheet and ListView has tiny scroll of one pixel

### LINKED ISSUE #98314: showModalBottomSheet with DraggableScrollableSheet and ListView has tiny scroll of one pixel
## Steps to Reproduce

1. Execute `flutter run` on the code sample 
2. Click Open Modal
3. Use the scroll wheel to scroll up and down

**Expected results:** The list view does not scroll since the content is smaller than the viewport

**Actual results:** The list view scrolls up and down by a single pixel

<details>
<summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: "Test",
      theme: ThemeData(),
      home: Scaffold(
        appBar: AppBar(
          title: const Text("Test"),
        ),
        body: Builder(
          builder: (context) {
            return ElevatedButton(
              child: const Text("Open Modal"),
              onPressed: () {
                showModalBottomSheet(
                  context: context,
                  isScrollControlled: true,
                  builder: (context) {
                    return DraggableScrollableSheet(
                      maxChildSize: 1.0,
                      initialChildSize: 1.0,
                      builder: (_, controller) {
                        return ListView.builder(
                          controller: controller,
                          itemBuilder: (context, i) {
                            return ListTile(
                              title: Text("Item $i"),
                            );
                          },
                          itemCount: 4,
                        );
                      },
                    );
                  },
                );
              },
            );
          },
        ),
      ),
    );
  }
}

```

</details>

<details>
  <summary>Logs</summary>

```
[  +61 ms] executing: uname -m
[  +30 ms] Exit code 0 from: uname -m
[        ] x86_64
[   +5 ms] executing: [/opt/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +6 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[        ] executing: [/opt/flutter/] git tag --points-at 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[  +12 ms] Exit code 0 from: git tag --points-at 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[        ] 2.10.0
[   +8 ms] executing: [/opt/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[   +7 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [/opt/flutter/] git ls-remote --get-url origin
[   +9 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +105 ms] executing: [/opt/flutter/] git rev-parse --abbrev-ref HEAD
[   +8 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +51 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +50 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[  +10 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +126 ms] Skipping pub get: version match.
[  +66 ms] Generating /home/nathan/workspace/test/audio_test/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[ +102 ms] Initializing file store
[  +12 ms] Skipping target: gen_localizations
[   +5 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
/home/nathan/workspace/test/audio_test/.dart_tool/package_config_subset}
[  +52 ms] gen_dart_plugin_registrant: Complete
[   +1 ms] Skipping target: _composite
[   +1 ms] complete
[   +5 ms] Launching lib/main.dart on Linux in debug mode...
[   +5 ms] /opt/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev /opt/flutter/bin/cache/artifacts/engine/linux-x64/frontend_server.dart.snapshot --sdk-root
/opt/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names --experimental-emit-debug-metadata
-DFLUTTER_WEB_AUTO_DETECT=true --output-dill /tmp/flutter_tools.GWGYBG/flutter_tool.FPGSYS/app.dill --packages
/home/nathan/workspace/test/audio_test/.dart_tool/package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation
--filesystem-scheme org-dartlang-root --initialize-from-dill build/c075001b96339384a97db4862b8ab8db.cache.dill.track.dill
--enable-experiment=alternative-invalidation-strategy
[  +34 ms] Building Linux application...
[  +25 ms] <- compile package:audio_test/main.dart
[   +2 ms] executing: [build/linux/x64/debug/] cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DFLUTTER_TARGET_PLATFORM=linux-x64 /home/nathan/workspace/test/audio_test/linux
[ +132 ms] -- Configuring done
[  +10 ms] -- Generating done
[  +18 ms] -- Build files have been written to: /home/nathan/workspace/test/audio_test/build/linux/x64/debug
[  +17 ms] executing: ninja -C build/linux/x64/debug install
[  +62 ms] ninja: Entering directory `build/linux/x64/debug'
[+17506 ms] [1/5] Generating /home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/libflutter_linux_gtk.so,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_engine.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_method_call.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_method_response.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_value.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/fl_view.h,
/home/nathan/workspace/test/audio_test/linux/flutter/ephemeral/flutter_linux/flutter_linux.h, _phony_
[  +14 ms] [  +69 ms] executing: uname -m
[   +2 ms] [  +39 ms] Exit code 0 from: uname -m
[   +2 ms] [        ] x86_64
[        ] [   +5 ms] executing: [/opt/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] [  +15 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] [        ] 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[   +1 ms] [        ] executing: [/opt/flutter/] git tag --points-at 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[        ] [  +12 ms] Exit code 0 from: git tag --points-at 5f105a6ca7a5ac7b8bc9b241f4c2d86f4188cf5c
[        ] [        ] 2.10.0
[        ] [   +7 ms] executing: [/opt/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[        ] [  +10 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] [        ] origin/stable
[   +1 ms] [        ] executing: [/opt/flutter/] git ls-remote --get-url origin
[   +2 ms] [   +5 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] [        ] https://github.com/flutter/flutter.git
[   +1 ms] [ +134 ms] executing: [/opt/flutter/] git rev-parse --abbrev-ref HEAD
[        ] [   +5 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] [        ] stable
[        ] [  +59 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] [   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms] [ +202 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms] [   +1 ms] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[        ] [ +109 ms] Initializing file store
[   +2 ms] [  +32 ms] Skipping target: gen_localizations
[        ] [  +12 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated contents:
/home/nathan/workspace/test/audio_test/.dart_tool/package_config_subset}
[        ] [   +5 ms] unpack_linux: Starting due to {}
[   +4 ms] [  +95 ms] unpack_linux: Complete
[        ] [ +766 ms] gen_dart_plugin_registrant: Complete
[        ] [        ] kernel_snapshot: Starting due to {}
[   +1 ms] [  +17 ms] /opt/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev /opt/flutter/bin/cache/artifacts/engine/linux-x64/frontend_server.dart.snapshot
--sdk-root /opt/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --target=flutter --no-print-incremental-dependencies -DFLUTTER_WEB_AUTO_DETECT=true
-Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation --packages
/home/nathan/workspace/test/audio_test/.dart_tool/package_config.json --output-dill
/home/nathan/workspace/test/audio_test/.dart_tool/flutter_build/28fad3f43840160d3b89c7dab0de1c46/app.dill --depfile
/home/nathan/workspace/test/audio_test/.dart_tool/flutter_build/28fad3f43840160d3b89c7dab0de1c46/kernel_snapshot.d package:audio_test/main.dart
[   +6 ms] [+13634 ms] kernel_snapshot: Complete
[   +4 ms] [ +448 ms] debug_bundle_linux-x64_assets: Starting due to {}
[        ] [ +170 ms] debug_bundle_linux-x64_assets: Complete
[        ] [ +541 ms] Persisting file store
[        ] [   +5 ms] Done persisting file store
[        ] [   +5 ms] build succeeded.
[        ] [   +9 ms] "flutter assemble" took 16,092ms.
[        ] [  +80 ms] ensureAnalyticsSent: 76ms
[        ] [        ] Running shutdown hooks
[        ] [        ] Shutdown hooks complete
[        ] [   +8 ms] exiting with code 0
[ +548 ms] [2/5] Building CXX object CMakeFiles/audio_test.dir/flutter/generated_plugin_registrant.cc.o
[ +149 ms] [3/5] Building CXX object CMakeFiles/audio_test.dir/my_application.cc.o
[ +246 ms] [4/5] Linking CXX executable intermediates_do_not_run/audio_test
[        ] [4/5] Install the project...
[   +6 ms] -- Install configuration: "Debug"
[  +24 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/audio_test
[        ] -- Set runtime path of "/home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/audio_test" to "$ORIGIN/lib"
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/icudtl.dat
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/lib/libflutter_linux_gtk.so
[  +35 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/AssetManifest.json
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/fonts
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[   +1 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/NOTICES.Z
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/FontManifest.json
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/kernel_blob.bin
[  +60 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/version.json
[   +1 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/packages
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/packages/cupertino_icons
[   +1 ms] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/packages/cupertino_icons/assets
[        ] -- Installing: /home/nathan/workspace/test/audio_test/build/linux/x64/debug/bundle/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[  +24 ms] Building Linux application... (completed in 18.9s)
[  +56 ms] (audio_test:2901512): Gtk-WARNING **: 00:09:58.889: Locale not supported by C library.
[        ]      Using the fallback 'C' locale.
[ +288 ms] Observatory URL on device: http://127.0.0.1:37099/FfukU0AUtyY=/
[   +5 ms] Caching compiled dill
[  +67 ms] Connecting to service protocol: http://127.0.0.1:37099/FfukU0AUtyY=/
[ +237 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at http://127.0.0.1:37099/FfukU0AUtyY=/.
[ +116 ms] DDS is listening at http://127.0.0.1:44673/FPHPcA2AcxE=/.
[ +106 ms] Successfully connected to service protocol: http://127.0.0.1:37099/FfukU0AUtyY=/
[  +98 ms] DevFS: Creating new filesystem on the device (null)
[  +34 ms] DevFS: Created new filesystem on the device (file:///tmp/audio_testMDGIGG/audio_test/)
[  +10 ms] Updating assets
[ +149 ms] Syncing files to device Linux...
[   +9 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[        ] Processing bundle.
[   +1 ms] <- recompile package:audio_test/main.dart 214c3013-85ea-44de-9244-2a9e1050736d
[        ] <- 214c3013-85ea-44de-9244-2a9e1050736d
[   +1 ms] Bundle processing done.
[ +140 ms] Updating files.
[        ] DevFS: Sync finished
[   +1 ms] Syncing files to device Linux... (completed in 148ms)
[   +5 ms] Synced 0.0MB.
[   +4 ms] <- accept
[   +8 ms] Connected to _flutterView/0x5593d0865d30.
[   +3 ms] Flutter run key commands.
[   +3 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h List all available interactive commands.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] 💪 Running with sound null safety 💪
[   +1 ms] An Observatory debugger and profiler on Linux is available at: http://127.0.0.1:44673/FPHPcA2AcxE=/
[ +196 ms] The Flutter DevTools debugger and profiler on Linux is available at: http://127.0.0.1:9101?uri=http://127.0.0.1:44673/FPHPcA2AcxE=/
[+4943 ms] Service protocol connection closed.
[   +2 ms] Lost connection to device.
[  +13 ms] DevFS: Deleting filesystem on the device (file:///tmp/audio_testMDGIGG/audio_test/)
[ +252 ms] Ignored error while cleaning up DevFS: TimeoutException after 0:00:00.250000: Future not completed
[   +7 ms] "flutter run" took 26,249ms.
[  +99 ms] ensureAnalyticsSent: 96ms
[        ] Running shutdown hooks
[        ] Shutdown hooks complete
[   +1 ms] exiting with code 0
```

```
Analyzing audio_test...                                                 
No issues found! (ran in 2.0s)
```

```
[✓] Flutter (Channel stable, 2.10.0, on Arch Linux 5.15.19-1-lts, locale en_US.UTF-8)
    • Flutter version 2.10.0 at /opt/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5f105a6ca7 (10 days ago), 2022-02-01 14:15:42 -0800
    • Engine revision 776efd2034
    • Dart version 2.16.0
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31)
    • Android SDK at /opt/android-sdk
    • Platform android-31, build-tools 31
    • ANDROID_HOME = /opt/android-sdk
    • Java binary at: /home/nathan/Documents/jdk-15.0.2/bin/java
    • Java version OpenJDK Runtime Environment (build 15.0.2+7-27)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /usr/bin/google-chrome-stable

[✓] Linux toolchain - develop for Linux desktop
    • clang version 13.0.0
    • cmake version 3.22.2
    • ninja version 1.10.2
    • pkg-config version 1.8.0

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions).

[✓] IntelliJ IDEA Community Edition (version 2021.3)
    • IntelliJ at /usr/share/idea
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] Connected device (3 available)
    • Pixel 5 (mobile) • 192.168.1.130:5555 • android-arm64  • Android 11 (API 30)
    • Linux (desktop)  • linux              • linux-x64      • Arch Linux 5.15.19-1-lts
    • Chrome (web)     • chrome             • web-javascript • Google Chrome 98.0.4758.80

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

</details>

example

### PR DESCRIPTION
**Pull request by xu-baolin**
Fixes #98314

I have change another solution to rework #31832 that remove the side affect.
@dnfield Could you take a look at this?

### CODE REVIEW

**Review by dnfield — COMMENTED** [review]
This seems fine, @Piinks do you mind taking a look as well?

File: packages/flutter/test/material/bottom_sheet_test.dart [hunk 1]
```diff
@@ -879,7 +879,6 @@ void main() {
                   children: <TestSemantics>[
                     TestSemantics(
                       flags: <SemanticsFlag>[SemanticsFlag.hasImplicitScrolling],
-                      actions: <SemanticsAction>[SemanticsAction.scrollDown, SemanticsAction.scrollUp],
```
Comment (Piinks) [inline packages/flutter/test/material/bottom_sheet_test.dart:882]: Why is this removed?

**Review by Piinks — COMMENTED** [review]
Hey @xu-baolin good to see you!

File: packages/flutter/test/material/bottom_sheet_test.dart (hunk 1 above)
Comment (xu-baolin) [reply packages/flutter/test/material/bottom_sheet_test.dart:882]: This is a line of reverting the original fixes. Giving one pixel content dimensions makes it scrollable.

Comment (Piinks) [reply packages/flutter/test/material/bottom_sheet_test.dart:882]: Ah ok, and so it serves as a regression test. Thanks for explaining!

**Review by Piinks — APPROVED** [review]
LGTM!

**Pull request merged**